### PR TITLE
Fixed transport description in minion / master config

### PR DESCRIPTION
**Pull request by jacobhammons**

### CODE REVIEW

**Comment by rallytime** [comment]
Thanks for grabbing those so fast @jacobhammons! I merged your first PR while @cachedout was commenting...

File: doc/ref/configuration/master.rst [hunk 1]
```diff
@@ -499,11 +499,11 @@ that connect to a master via localhost.
 ``transport``
 -------------
 
-Default: ``zeromq``
+Default: ``ZeroMQ``
```
Comment (rallytime) [inline doc/ref/configuration/master.rst]: So these defaults as a code setting should remain ``zeromq``. It's the proper noun at the beginning of the second sentence in the description that should be ``ZeroMQ``.

File: doc/ref/configuration/minion.rst [hunk 2]
```diff
@@ -583,11 +583,11 @@ Pull port used when :conf_minion:`ipc_mode` is set to ``tcp``.
 ``transport``
 -------------
 
-Default: ``zeromq``
+Default: ``ZeroMQ``
```
Comment (rallytime) [inline doc/ref/configuration/minion.rst]: Same thing in this file. Code settings should still be ``zeromq``, but used in a sentence should be ``ZeroMQ``.

**Comment by jacobhammons** [comment]
@rallytime that makes more sense!

**Comment by rallytime** [comment]
Perfect!

**Pull request merged**